android/filament-android/src/main/cpp/Stream.cpp

```diff
@@ -189,8 +184,15 @@ Java_com_google_android_filament_Stream_nSetAcquiredImage(JNIEnv* env, jclass, j
 
 #ifdef ANDROID
 
-    AHardwareBuffer* (*AHardwareBuffer_fromHardwareBuffer)(JNIEnv*, jobject);
-    loadSymbol(AHardwareBuffer_fromHardwareBuffer, "AHardwareBuffer_fromHardwareBuffer");
+    using PFN_FROMHARDWAREBUFFER = AHardwareBuffer* (*)(JNIEnv*, jobject);
+    static PFN_FROMHARDWAREBUFFER AHardwareBuffer_fromHardwareBuffer =
```

Comment:
Just be aware that when you do this, the compiler inserts a mutex around the test/assignment (because it can't know that this is not called from multiple threads). It's probably not a big deal here.
You could make it a bit better by moving the `static` outside of the function and testing if it's `nullptr` yourself -- assuming you know this is only called from a single thread.